tools/dependencies/openblas.sh

```diff
@@ -33,7 +33,12 @@ if [[ ((! -e $DEPS_PATH/lib/libopenblas.a) && -z "$CMAKE_STATICBUILD") ||
     cd $DEPS_PATH/OpenBLAS-$OPENBLAS_VERSION
 
     # Adding NO_DYNAMIC=1 flag causes make install to fail
-    CXX="g++ -fPIC" CC="gcc -fPIC" $MAKE DYNAMIC_ARCH=1 USE_OPENMP=1
+    if [[ ! $ARCH == 'aarch64' ]]; then
+        CXX="g++ -fPIC" CC="gcc -fPIC" $MAKE DYNAMIC_ARCH=1 USE_OPENMP=1
+    else
+        # openblas build fails with gcc-8.5, hence using gcc-8.4 explicitly
+        CXX="g++-8 -fPIC" CC="gcc-8 -fPIC" FC="gfortran-8" $MAKE DYNAMIC_ARCH=1 USE_OPENMP=1
```

Comment:
Have you tried updating openblas?

Reply:
Tried OpenBLAS v0.3.10 (same version as master) and it works with gcc-8.5 , updated it. Thanks for the suggestion.

Reply:
Great! Would you consider v0.3.15? I see there are a couple of arm fixes in between of v0.3.10 and v0.3.15 including "Fixed segmentation faults in DYNAMIC_ARCH builds" which would affect us as we use DYNAMIC_ARCH=1